test.js

```diff
@@ -134,3 +142,26 @@ test('passthrough options', validate, {
 	stopEarly: true,
 	unknown: true
 });
+
+test('fail if type is not boolean|string|number', fail, {
```

Comment:
Could you not use macros for these tests? Macros looks great when it's strictly testing input/output.